cli/types/cypress.d.ts

```diff
@@ -2353,8 +2353,13 @@ declare namespace Cypress {
      */
     reporter: string
     /**
-     * Whether to take a screenshot on test failure when running headlessly or in CI
-     * @default true
+     * Some reporters accept [reporterOptions](https://on.cypress.io/reporters) that customize their behavior
+     * @default "spec"
+     */
+    reporterOptions: { [key: string]: any }
+    /**
+     * Customize reporter behavior by passing options
+     * @default null
```

Comment:
Actually, this description is for `watchForFileChanges` below and has been incorrect for a while apparently. 

```suggestion
     * Whether Cypress will watch and restart tests on test file changes
     * @default true
```